Composer/README.md

```diff
@@ -3,13 +3,15 @@
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/microsoft/BotFramework-Composer.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/microsoft/BotFramework-Composer/alerts/)
 
 # Composer
+
 The web app that can edit bots in OBI format, and can use Bot Launcher to run bot.
 
 ### Instructions
 
 Prerequisite:
-* node > 12
-* yarn         // npm install -g yarn
+
+- node > 14
```

Comment:
Our required is only [BotFramework-Composer/package.json at main · microsoft/BotFramework-Composer (github.com)](https://github.com/microsoft/BotFramework-Composer/blob/main/Composer/package.json#L24) node 12. A constraint imposed by bf packages all which require only node 12.